andela/ah-frontend-space
# #166880075 Users should be able to Login to Authors Haven

#### What does this PR do?

- Adds login functionality to the application

#### Description of Task to be completed?

- Add login form, email input, and password input components.
- Pick data from the login form and make a request to the `ah-backend-API` to log in a user.
- Redirect the user to the `articles` page upon successful login or display an error in case it fails

#### How should this be manually tested?

- Go to the [sign in](https://ah-frontend-space-stagin-pr-15.herokuapp.com/signin) page 
- Enter your email and password in the fields then enter 
- If your account is valid, you will be redirected to the `articles` page
NOTE: A registered account is required to access this route - you can register with the API on this [link](https://ah-haven-space.herokuapp.com/#api-users-create). Ensure to use a valid email address

#### What are the relevant pivotal tracker stories?

[#166880075](https://www.pivotaltracker.com/story/show/166880075)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/6644707/61435063-50328500-a940-11e9-8af5-77a5acc3c967.png)
